nco/.htaccess

```
@@ -9,4 +9,4 @@ RewriteEngine on
 #RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
 RewriteRule ^$ https://raw.githubusercontent.com/ProfTuan/NCO/refs/heads/main/nco.owl [R=302,L]
 
-RewriteRule ^/merged$ https://raw.githubusercontent.com/ProfTuan/NCO/refs/heads/main/nco-merged.owl [R=302,L]
\ No newline at end of file
+RewriteRule ^nco/merged$ https://raw.githubusercontent.com/ProfTuan/NCO/refs/heads/main/nco-merged.owl [R=302,L]
```

Paths are relative.  You probably want this:
```suggestion
RewriteRule ^merged$ https://raw.githubusercontent.com/ProfTuan/NCO/refs/heads/main/nco-merged.owl [R=302,L]
```